cg/apps/downsample/downsample.py

```diff
@@ -0,0 +1,169 @@
+"""API that handles downsampling of samples."""
+import logging
+from pathlib import Path
+
+from cg.apps.downsample.utils import case_exists_in_statusdb, sample_exists_in_statusdb
+from cg.apps.housekeeper.hk import HousekeeperAPI
+from cg.constants import SequencingFileTag
+from cg.exc import DownsampleFailedError
+from cg.meta.meta import MetaAPI
+from cg.meta.workflow.downsample.downsample import DownsampleWorkflow
+from cg.models.cg_config import CGConfig
+from cg.models.downsample.downsample_data import DownsampleData
+from cg.store import Store
+from cg.store.models import Family, FamilySample, Sample
+from cg.utils.calculations import multiply_by_million
+
+LOG = logging.getLogger(__name__)
+
+
+class DownsampleAPI(MetaAPI):
+    def __init__(
+        self,
+        config: CGConfig,
+        sample_id: str,
+        number_of_reads: float,
+        case_id: str,
+        dry_run: bool = False,
+    ):
+        """Initialize the API."""
+        super().__init__(config)
+        self.config: CGConfig = config
+        self.status_db: Store = config.status_db
+        self.hk_api: HousekeeperAPI = config.housekeeper_api
```

Comment:
These can be removed since they are defined in the base class `MetaAPI` already

Reply:
That is true, will fix it